src/app/tabs/run-tab.js

```diff
@@ -581,6 +583,29 @@ function settings (container, self) {
       }
     })
   }
+  function signMessage (event) {
```

Comment:
we might need to modify this function so it it also working with Web3:
if the current env is `JavaScript VM` use the current behavior.
if the current env is not JavaScript VM and if personal mode is enabled `self._deps.config.get('settings/personal-mode')`, 
use the following function to sign: web3.personal.sign(message, account, password) 
https://github.com/ethereum/go-ethereum/wiki/Management-APIs#personal_sign
(the ui then need to ask for the password - using https://github.com/ethereum/remix-ide/blob/master/src/app/ui/modal-dialog-custom.js#L12)

Reply:
@yann300 I tried to use the personal_sign RPC call from web3 but now Metamask is not responding to any calls to get the account list or anything, similar to this issue here.
https://ethereum.stackexchange.com/questions/52197/metamask-remix-ide-cannot-create-an-account-error-internal-json-rpc-error
Any ideas as to what could be the root cause of the problem?
FYI, I noticed the personal module in exectionContext.web3().personal doesnt seem to contain a sign function. Then something I did caused this issue to occur.